### PR TITLE
feat: add drill link example to tile sdk demo

### PR DESCRIPTION
**Pull request by bryans99**

### CODE REVIEW

File: react/javascript/tile-sdk/src/components/Inspector/components/EventTester/EventTester.js [hunk 1]
```diff
@@ -97,11 +99,31 @@ export const EventTester = () => {
   )
 
   const openDrillMenuClick = useCallback(
-    (event) => {
-      // TODO links data needs to be populated
-      tileSDK.openDrillMenu({ links: [] }, event)
+    (_event) => {
+      let event = { pageX: 0, pageY: 0 }
+      let links = []
+      if (visualizationData?.queryResponse?.data) {
```
Comment (bryans99) [inline react/javascript/tile-sdk/src/components/Inspector/components/EventTester/EventTester.js:105]: TODO - add utility methods to the extension SDK
1. get all drill links for column name - returns an array of drill links for the specified column name
2. 1. get drill links for column name and row index - returns drill links for the specified column name and row index

File: react/javascript/tile-sdk/src/components/Inspector/components/EventTester/EventTester.js [hunk 2]
```diff
@@ -97,11 +99,31 @@ export const EventTester = () => {
...
+        const data = visualizationData?.queryResponse?.data
```
Comment (jkaster) [inline react/javascript/tile-sdk/src/components/Inspector/components/EventTester/EventTester.js:106]: very minor nit but couldn't you assign this outside the `if` and then check it for the conditional logic?

**Review by jkaster — APPROVED** [review]

File: react/javascript/tile-sdk/src/components/Inspector/components/EventTester/EventTester.js (hunk 2 above)
Comment (bryans99) [reply react/javascript/tile-sdk/src/components/Inspector/components/EventTester/EventTester.js]: Yep, I like that! Done

**Review by jkaster — APPROVED** [review]
LGTM

**Pull request merged**